### PR TITLE
[fix] packaging: comply with 2.4 multi-instance

### PR DESCRIPTION
**Pull request by JocelynDelalande**
No longer using last-argument in favor of $YNH_APP_INSTANCE_NAME.

### CODE REVIEW

File: packaging_apps_multiinstance.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 Multi-instance is application capacity to be installed several times.
 
 #### Scripts
-When YunoHost installs the application a second time, it gives to the script, at last argument, `${id}__2` with the application `${id}` coming from the manifest. The value `n` in `${id}__n` is incremented each time a new instance of the application is installed.
+When YunoHost installs the application a second time, it pass `$YNH_APP_INSTANCE_NAME` var to the script, set to value `${id}__2` with the application `${id}` coming from the manifest. The value `n` in `${id}__n` is incremented each time a new instance of the application is installed.
```
Comment (opi) [inline packaging_apps_multiinstance.md]: $YNH_APP_INSTANCE_NAME is passed every time, not only the second (or more) time YunoHost installs the app.

Maybe we should talk about $YNH_APP_ID and  $YNH_APP_INSTANCE_NUMBER here

Comment (JocelynDelalande) [inline packaging_apps_multiinstance.md]: Fixed, thanks :)

File: packaging_apps_multiinstance.md [hunk 2]
```diff
@@ -4,13 +4,13 @@
 Multi-instance is application capacity to be installed several times.
 
 #### Scripts
-When YunoHost installs the application a second time, it gives to the script, at last argument, `${id}__2` with the application `${id}` coming from the manifest. The value `n` in `${id}__n` is incremented each time a new instance of the application is installed.
+When YunoHost installs the application, it pass `$YNH_APP_INSTANCE_NAME` var to the script, set to value `id__n` with the application `id` coming from the manifest and `n` being an integer  incremented each time a new instance of the application is installed.
```
Comment (julienmalik) [inline packaging_apps_multiinstance.md]: it passes [...], sets

Comment (JocelynDelalande) [inline packaging_apps_multiinstance.md]: @julienmalik agreed for "passes", not for "sets" (it's a present participle, not a conjugated verb).

Comment (julienmalik) [inline packaging_apps_multiinstance.md]: agreed, I looked too fast :)

**Pull request merged**